Dockerfile

```diff
@@ -1,14 +1,36 @@
-FROM docker.io/library/node:16.14.2 AS build
+FROM registry.redhat.io/ubi8/nodejs-16:1-72 AS web-builder
 
-WORKDIR /usr/src/app
+WORKDIR $REMOTE_SOURCES_DIR/logging-view-plugin
```

Comment:
`REMOTE_SOURCES_DIR` is a cpaas thing that is populated when it generates a Dockerfile.  You either need to set the actual path here or define an ENV var